app/controllers/patients_controller.rb

```diff
@@ -31,10 +31,8 @@ def show
 
     # All household members regardless if current patient is HOH
     household = current_user.get_patient(@patient.responder_id)&.dependents
-    @household_members = ([@patient] + (household.nil? ? [] : household)).uniq
-
-    # All household members that are in the exposure workflow with continuous exposure excluding the current patient
-    @household_members_with_ce_in_exposure_excludes_patient = household.nil? ? [] : household.where(isolation: false, continuous_exposure: true)
+    @household_members = (household.nil? ? [] : household.where(purged: false)).uniq
+    @household_members_exclude_self = (household.nil? ? [] : household.where.not(id: @patient.id).where(purged: false)).uniq
```

Comment:
Could you make use of https://github.com/SaraAlert/SaraAlert/blob/master/app/models/patient.rb#L641 to simplify this logic? Also not sure uniq is needed (I understand it is an artifact of how this looked below).

Reply:
updated here: https://github.com/SaraAlert/SaraAlert/pull/722/commits/9d58d8d4f855e244bb449310e65e3d92a4657f35